azure-kusto-data/tests/e2e.py

```diff
@@ -300,3 +303,33 @@ def test_cloud_info(self):
     def test_cloud_info_404(self):
         cloud_info = CloudSettings.get_cloud_info_for_cluster("https://www.microsoft.com")
         assert cloud_info is CloudSettings.DEFAULT_CLOUD
+
+    def test_redirects(self):
+        redirect_codes = [301, 302, 303, 307, 308]
+        with KustoClient("https://httpstat.us/") as client:
```

Comment:
How does this pass the DNS checks?
Should we not set an allow policy somewhere for this URI?

Reply:
The logic currently validates by cloud, since we use no token provider, we don't fetch cloud data, so we can't check the endpoints. I don't know if it's good or bad, but that's the logic.